File: Pathfinder.Web/ClientApp/src/models/Race.js
class Race {
  constructor() {
    this.id = null;
    this.name = null;
    this.sizeId = null;
    this.baseSpeed = null;
    this.size = null;
  }

  static init(object) {
    let race = new Race();
    race.id = object.id;
    race.name = object.name;
    race.sizeId = object.sizeId;
    race.baseSpeed = object.baseSpeed;
    race.size = object.size;
    return race;
  }
}

File: CharacterManagement.Domain/Entity/Race.cs
namespace Pathfinder.CharacterManagement.Domain.Entity;

public class Race: Pathfinder.Utils.Entities.Base.Entity
{
    public string Name { get; set; }
    public RaceSizeType SizeId { get; set; }
    //public virtual RaceSize Size { get; set; }
    public int BaseSpeed { get; set;}
    public bool IsNightVision { get; set; }
}